### DISCUSSION TITLE
Build with Release instead of Debug?

### DISCUSSION THREAD

**Discussion by laoshaw** [aws/aws-sdk-cpp#2002] [Q&A] [answered]
the document says to build with Debug, should I build with Release instead as I have no intention to debug the SDK itself other than use it to develop my code. Besides, Debug build has size problems.

why is the README recommending a Debug build for its users?

**Comment by laoshaw** [comment]
Release build works, Thanks.

**Comment by jmklix** [comment, marked as answer]
If you run into any errors with the sdk you will probably want debug so you can get more useful error messages. It will still work with release as you noticed.
